download-latest.sh

```diff
@@ -65,6 +65,30 @@ get_os() {
     return 0
 }
 
+# Gets the C lib the system is using by setting the $c_lib variable.
```

Comment:
we don't use `$c_lib` in the script, so no point in storing it.
Can you remove it?